File: CMS-API/DTOs/CarManageRecordDto.cs
using System;

namespace CMS_API.DTOs
{
    public class CarManageRecordDto
    {
        public string CompanyName { get; set; }
        public string PlateNumber { get; set; }
        public string DriverName { get; set; }
        public string LicenseNumber { get; set; }
        public DateTime SignInDate { get; set; }


        public string TempNumber { get; set; }
        public string SignInReason { get; set; }
        public string GoodsName { get; set; }
        public string GoodsCount { get; set; }
        public string DepartmentName { get; set; }

        public string ContactPerson { get; set; }
        public string SealNumber { get; set; }
        public string DriverSign { get; set; }
        public DateTime? SignOutDate { get; set; }
        public string GuardName { get; set; }

        public string CardSize { get; set; }
        public int CompanyDistance { get; set; }
    }
}

File: CMS-SPA/src/app/core/_models/car-manage-record-dto.ts
import { ModelInterface } from "./interface/model-interface";

export class CarManageRecordDto implements ModelInterface {
  companyName: string;
  plateNumber: string;
  driverName: string;
  licenseNumber: string;
  signInDate: Date;

  tempNumber: string;
  signInReason: string;
  goodsName: string;
  goodsCount: string;
  departmentName: string;

  contactPerson: string;
  sealNumber: string;
  driverSign: string;
  signOutDate: Date;
  guardName: string;

  carSize: string;
  companyDistance: number;

  isDisplay: number; //only use in front end
  constructor(){}
}
